<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_client</class>
 <widget class="QMainWindow" name="Add_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibSys</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>261</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>21</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DODAJ KLIENTA</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Verdana</family>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5"/>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Verdana</family>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Imie</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="client_first_name">
           <property name="placeholderText">
            <string>Imie</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="client_last_name">
           <property name="placeholderText">
            <string>Nazwisko</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Verdana</family>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nazwisko</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="client_email">
           <property name="placeholderText">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Verdana</family>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Data urodzenia</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="client_date_of_birth"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Verdana</family>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tel.</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="client_phone">
           <property name="placeholderText">
            <string>Numer telefonu</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="PushButton_add_client">
         <property name="text">
          <string>Dodaj klienta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>511</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="search_client_lineedit">
       <property name="placeholderText">
        <string>Wyszukaj</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="client_table_view">
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="client_delete_button">
           <property name="font">
            <font>
             <family>Verdana</family>
            </font>
           </property>
           <property name="text">
            <string>Usu≈Ñ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="client_history_button">
           <property name="font">
            <font>
             <family>Verdana</family>
            </font>
           </property>
           <property name="text">
            <string>Historia</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
